gems/aws-sdk-core/lib/aws-sdk-core/checksums.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'tempfile'
+require 'digest'
+require_relative 'checksums/digest'
+require_relative 'checksums/digest32'
+require_relative 'checksums/md5'
+require_relative 'checksums/sha1'
+require_relative 'checksums/sha256'
+require_relative 'checksums/crc32'
+require_relative 'checksums/crc32c'
+
+module AWS::SDK::Core
+  # A utility module for calculating checksums.
```

Comment:
It doesn't calculate any checksums though.. I'd say this is just a namespace for checksum classes. Yes there's one utility method but I wouldn't say it's the reason we have this module.

Reply:
Yeah - I think we can potentially remove this and move the checksum algorithm method to the checksum middleware.

I'd still like a nice place for all of the digest classes to live, and I'm not sure where that should be though.  I guess right now the checksum middleware is the only thing using them so they could just all be classes within that?